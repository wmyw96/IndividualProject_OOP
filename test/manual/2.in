..2###...6
2#.###3...
5......5..
...####...
...#6.....
.....#####
3..4.#####
..4###...1
...#......
...#1.....
