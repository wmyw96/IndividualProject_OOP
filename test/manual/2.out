Maximize the total connected pairs: 5 pairs
222***..66
2*.***..6.
555555556.
...****66.
...*6666..
.....*****
..44.*****
..4***1111
...*.11...
...*11....